src/i18n/en-US/index.ts

```diff
@@ -352,7 +352,7 @@ export default {
       requiredClaimFirst: 'Please claim your rewards before sending transaction',
       requiredClaimFirstCompounding:
         '{message} -Disable compounding, claim your rewards and then enable compounding again',
-      successfullyWithdrew: 'Balance is successfully withdrew',
+      successfullyWithdrawn: 'Balance is successfully withdrawn',
```

Comment:
Not a blocker, but maybe drop "is" for brevity, so "Balance successfully withdrawn"
```suggestion
      successfullyWithdrawn: 'Balance successfully withdrawn',
```

Reply:
Thanks, fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/928/commits/9706be51148fa5561727a6ef0dcbee83a805d46e) commit.